js/src/ramsey-batch.js

```diff
@@ -61,18 +62,23 @@ jQuery( document ).ready( ( $ ) => {
 				method: 'POST',
 				data: {
 					action: 'ramsey-batch-item',
-					item
+					item,
+					batchName: ramseyBatch.batchName
 				},
 				success( response ) {
 					if ( ! response.success ) {
-						console.log( response.data.reason, response );
+						console.error( response.data.reason, response );
 
 						return reject();
 					}
 
 					ramseyBatch.itemsComplete++;
 
-					console.log( response.data.reason, response );
+					if ( 'warn' == response.data.type ) {
```

Comment:
This would be trippy, but would save you a couple of lines and an `if` statement.

```js
console.[response.data.type](response.data.reason, response);
```

Reply:
Going to hold off on this as I want to ensure that only the type of 'warn' changes anything.